Sources/StreamVideo/Controllers/CallController.swift

```diff
@@ -74,7 +73,6 @@ class CallController: @unchecked Sendable {
         self.callType = callType
         self.apiKey = apiKey
         self.videoConfig = videoConfig
-        sfuReconnectionTime = 30
```

Comment:
this wasn't used anywhere, right?

Reply:
That's right. That variable has been unused since we move to the new WebRTC stack where reconnection time is handled [here](https://github.com/GetStream/stream-video-swift/blob/95988c30a1a088679d1fc28972259b0657f2c833/Sources/StreamVideo/WebRTC/v2/StateMachine/Stages/WebRTCCoordinator%2BJoining.swift#L377).

Reply:
I believe we fixed this already, we should merge develop.